tests/test_transpilers_placer.py

```diff
@@ -0,0 +1,158 @@
+import networkx as nx
+import pytest
+from qibo import gates
+from qibo.models import Circuit
+
+from qibolab.transpilers.placer import (
+    Backpropagation,
+    Custom,
+    Random,
+    Subgraph,
+    Trivial,
+    assert_mapping_consistency,
+    assert_placement,
+)
+
+
+def star_connectivity():
+    Q = ["q" + str(i) for i in range(5)]
+    chip = nx.Graph()
+    chip.add_nodes_from(Q)
+    graph_list = [
+        (Q[0], Q[2]),
+        (Q[1], Q[2]),
+        (Q[3], Q[2]),
+        (Q[4], Q[2]),
+    ]
+    chip.add_edges_from(graph_list)
+    return chip
+
+
+def star_circuit():
+    circuit = Circuit(5)
+    for i in range(1, 5):
+        circuit.add(gates.CNOT(i, 0))
+    return circuit
+
+
+def test_assert_placement_true():
+    layout = {"q0": 0, "q1": 1, "q2": 2, "q3": 3, "q4": 4}
+    circuit = Circuit(5)
+    assert assert_placement(circuit, layout, verbose=True)
+
+
+@pytest.mark.parametrize("qubits", [5, 3])
+@pytest.mark.parametrize("layout", [{"q0": 0, "q1": 1, "q2": 2, "q3": 3}, {"q0": 0, "q0": 1, "q2": 2}])
+def test_assert_placement_false(qubits, layout):
+    circuit = Circuit(qubits)
+    assert not assert_placement(circuit, layout, verbose=True)
```

Comment:
Pay attention, here you really need to do something different:
```suggestion
	with pytest.raises(PlacementError):
        assert_placement(circuit, layout, verbose=True)
```